service/src/main/java/edu/tamu/catalog/service/CatalogService.java

```diff
@@ -18,6 +20,8 @@
 
     HoldingsRecord getHolding(String id, String holdingId);
 
+    FeesFines getFeesFines(String uin) throws ParseException;
```

Comment:
Likely will want an exception handler for ParseException. I think 500 makes sense because this exception will be thrown due to a bad response from FOLIO.

Reply:
This already returns 500 without adding any handlers (pretty much every unhandled exception returns HTTP 500).
I have just confirmed this behavior.
Granted, it is the "Whitelabel Error Page".
The HTTP response code is still 500.

Reply:
That is the problem. It should be handled gracefully. Log stacktrace only in debug and return 500 with the message as application/json or plain/text.

Reply:
The REST API responses should be informative and in this case should be that a parsing exception occurred and is from a response from FOLIO.